### DISCUSSION TITLE
Installed Laragon - php files are being downloaded / not executed properly

### DISCUSSION THREAD

**Discussion by cb911nr** [leokhoa/laragon#821] [Q&A]
Any ideas? Fresh laragon installation, got a wordpress install, but when you visit /whatever.php it just downloads the file...

Just wanted to see if this was a common issue sometimes, and if there's an easy way to fix?

For example, installed xampp as well and that worked fine, so just confused on what could be the issue.

**Comment by Pen-y-Fan** [comment]
No, it's not a common issue. Although one person posted the same issue #819 , which was related to Apache version 2.4.60.

Normally a fresh installation of Laragon should have no issues.

When you install a project in the laragon/www directory, it will create an Apache conf file automatically and update your Windows host file. Sometimes you need to give elevated permissions for Laragon to make changes to these files. You also need to stop and start Laragon for the config changes to take effect. The project will be available using a pretty url like http://project-name.test. The project name will be the same as the folder that contains this project. See the documentation on [quick start](https://laragon.org/docs/quick-start) for more details.

Issues can occur due to various reasons, caused by something on your computer, such as Windows updates or other system changes that you may be unaware about. Anti-virus software sometimes has false positives. 

Reboot your PC to release file locks, check for Windows updates and install them.

If that doesn't fix the issue, view anti-virus logs for programs being prevented from running, directories created etc.

**Comment by rpascolo** [comment]
httpd-2.4.61-240703-win64-VS17

**Comment by MGeurts** [comment]
I can confirm. The version httpd-2.4.61-240703-win64-VS17 works as expected.
